README.md

```diff
@@ -1,77 +1,136 @@
-# Template for `Application Examples`
+# Template for `Application Examples` on GitHub
 
 ## What is an application example
 
-An application example ist a stand alone project, which demonstrates the usage of one ore more features or libraries. This template is optimized for a creation of application examples for PLC applications (executable on a PLCSIM Advanced or real PLC S7-1500).
+An application example is a standalone project, which demonstrates the usage of one ore more features or libraries. This template is optimized for a creation of application examples for PLC applications (executable on a PLCSIM Advanced or real PLC S7-1500).
 
-## Create a project from this template
+## Create a project from an "apax-template"
 
-Example: If you want to create an application example `ae-motion-control` then use this template by entering:
+If you want to create a project based from an apax-template you have to enter the following-scheme in the terminal :
 
 ```bash
-apax create @simatic-ax/app --registry https://npm.pkg.github.com ae-motion-control
+apax create @<registry-scope>/<template-name> --registry <npm-package-registry-url> <your-new-project-name>
```

Comment:
Also don't agree. The user need definitely the  @simatix-ax/app and the registry url. That means the user should simply copy the command and paste it into the client to start with the application.

Reply:
The command for the concrete copy is described down the line of that readme ... but I agree. To have a quick "copy" section at the beginning is more beneficial.